pavelib/assets.py

```diff
@@ -765,12 +765,22 @@ def webpack(options):
     Run a Webpack build.
     """
     settings = getattr(options, 'settings', Env.DEVSTACK_SETTINGS)
+    static_root_lms=Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
+    static_root_cms=Env.get_django_setting("STATIC_ROOT", "cms", settings=settings)
+    config_path=Env.get_django_setting("WEBPACK_CONFIG_PATH", "lms", settings=settings)
     environment = 'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
         node_env="production" if settings != Env.DEVSTACK_SETTINGS else "development",
```

Comment:
Definitely need to be clear when we're using prod/dev - I had originally intended "`envs/aws.py` gets prod, everyone else is dev", but this seems to indicate something different.